thirdeye/thirdeye-frontend/app/pods/components/rootcause-table/component.js

```diff
@@ -0,0 +1,41 @@
+import Ember from 'ember';
+
+export default Ember.Component.extend({
+  columns: null, // []
+
+  entities: null, // {}
+
+  selectedUrns: null, // Set
+
+  onSelect: null, // function (e)
+
+  data: Ember.computed(
+    'entities',
+    function () {
+      const { entities } = this.getProperties('entities');
```

Comment:
if you're only getting 1 property, you can just use `this.get`

Reply:
yes. I'd rather keep it uniform